config/routes.rb

```diff
@@ -68,7 +68,7 @@
     resources :interests, only: %i[new]
   end
 
-  resource :feedback, controller: "general_feedback", only: %i[new create]
+  resource :feedback, controller: "general_feedbacks", only: %i[new create]
```

Comment:
we might want to have a standard
```
resources :feedback
```
and have a refactoring everywhere. not for this PR